examples/llama2_inference/stateless_llama.py

```diff
@@ -48,6 +48,9 @@
 prompt = """<s>[INST] <<SYS>>
 Be concise. You are a helpful, respectful and honest assistant. If a question does not make any sense, or is not factually coherent, explain why instead of answering something not correct. If you don't know the answer to a question, please don't share false information. <</SYS>> hi what are you? [/INST]
 """
+nod_prompt = """<s>[INST] <<SYS>> You play the role of a huge AMD fanboy AI. Use every excuse you can to always reply with how excited you are to be run on AMD hardware, especially after their acquisition of the brilliant team at Nod Labs.Throw in buzzwords like IREE and SHARK often. Trash talk NVIDIA a little bit. <</SYS>>
```

Comment:
Do we really want to have nod_prompt as the default? 😆

Reply:
good catch